sporkfish/searcher/minimax.py

```diff
@@ -64,6 +65,7 @@ def __init__(
             logging.info("Disabled transposition table in search.")
 
         self._evaluator = evaluator
+        self.max_depth = searcher_config.max_depth
```

Comment:
i guess i should change max_depth to _max_depth?

Reply:
Yes pls